src/components/games/quiz/Quiz.test.tsx

```diff
@@ -23,317 +16,282 @@ const mockContent = {
+          answer: "A wrong answer"
         },
         {
-          answer: "Another wrong answer",
+          answer: "Another wrong answer"
         },
         {
-          answer: "Yet Another wrong answer",
+          answer: "Yet Another wrong answer"
         },
         {
           answer: "Correct answer",
-          isCorrect: true,
-        },
-      ],
-    },
-  ],
+          isCorrect: true
+        }
+      ]
+    }
+  ]
 };
-let user : UserEvent;
-
-beforeEach(() => {
-  jest.spyOn(FirebaseAPI, "fetchQuizData").mockImplementation(() => Promise.resolve(mockContent));
-});
-
-afterEach(() => {
-  cleanup();
-});
-
-async function clickCorrectAnswer() {
-  await user.click(screen.getByText("Correct answer"));
-}
-
-async function clickIncorrectAnswer() {
-  await user.click(screen.getByText("A wrong answer"));
-}
-
-async function clickNext() {
-  await user.click(screen.getByTestId("next-button"));
-}
-
-async function playQuizWithScore(score: number) {
-  for (let index = 0; index < score; index++) {
-    await clickCorrectAnswer().then(() => clickNext());
-  }
-  for (let index = score; index < mockContent.questions.length; index++) {
-    await clickIncorrectAnswer().then(() => clickNext());
-  }
-}
 
 describe("Quiz", () => {
-  test("navigate to quiz", async() => {
-    user = userEvent.setup();
-    // Define width so carousel on homepage loads
-    Object.defineProperties(window.HTMLElement.prototype, {
-      offsetWidth: {
-        get: () => 100,
-      },
-    });
-
-    const home = render(
-      <MemoryRouter>
-        <App />
-      </MemoryRouter>,
-    );
-
-    const games = home.getByTestId("Games");
-    expect(games).toBeInTheDocument();
-    await user.click(games);
-
-    const quiz = screen.getByTestId("Quiz");
-    expect(quiz).toBeInTheDocument();
-    await user.click(quiz);
-
-    const begin = screen.getByTestId("game-begin");
-    expect(begin).toBeInTheDocument();
-  });
-
   describe("Quiz functionality", () => {
     beforeEach(async () => {
-      user = userEvent.setup();
-      const quizRoute = "/Games/Quiz";
-
-      await act(async () => {
-        await render(
-          <RoutingTestWrapper path={quizRoute}>
-            <Route
-              path="/Games/Quiz"
-              element={(
-                <GameContext>
-                  <QuizContext>
-                    <Quiz />
-                  </QuizContext>
-                </GameContext>
-              )}
-            />
-          </RoutingTestWrapper>,
-        );
-      });
-      await user.click(screen.getByTestId("game-begin"));
+      jest
+        .spyOn(FirebaseAPI, "fetchQuizData")
+        .mockImplementation(() => Promise.resolve(mockContent));
     });
 
-    test("First question renders", () => {
-      expect(screen.getByText("First question")).toBeInTheDocument();
-    });
+    test("should alert whether answer was correct", async () => {
+      const user = userEvent.setup();
+
+      render(
```

Comment:
This is repeated in every test so could go in a beforeEach, but I think it is nice to be explicit in each test what is being rendered.